recipe/meta.yaml

```
@@ -1,28 +1,28 @@
 {% set name = "marshmallow" %}
-{% set version = "3.19.0" %}
+{% set version = "4.0.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 90032c0fd650ce94b6ec6dc8dfeb0e3ff50c144586462c389b81a07205bedb78
+  sha256: 3b6e80aac299a7935cfb97ed01d1854fb90b5079430969af92118ea1b12a8d55
 
 build:
-  skip: true  # [py<37]
+  skip: true  # [py<39]
   number: 0
   script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation . -vv
 
 requirements:
   host:
     - python
     - pip
-    - setuptools
-    - wheel
+    - flit-core
```

```suggestion
    - flit-core <4
```
https://github.com/marshmallow-code/marshmallow/blob/4.0.0/pyproject.toml#L50C23-L50C25

---

It built and tested locally with downstream packages added.

---

I added tickets for marshmallow-sqlalchemy  and marshmallow-oneofschema whose release notes state marshmallow 4 compatibility in recent releases.